build.gradle.kts

```diff
@@ -209,6 +209,7 @@ dependencies {
   implementation("com.redis.om:redis-om-spring:${redisOMVersion}")
   implementation("com.redis.testcontainers:testcontainers-redis-junit:$testcontainersRedis")
   implementation("org.springframework.boot:spring-boot-starter-test")
+  implementation("org.springframework.boot:spring-boot-starter-webflux")
```

Comment:
Could you specify why you add WebFlux as dependency ?
In LokiService, you use it like a simple http client (or RestTemplate). 
It seems overkill for a simple get request, a simple httpclient could do the same, except if I miss something

Reply:
For instance, we don't need it and we will not use reactive mecanism except if we have reasons to.
So you can remove this dependencies and use simple http client instead ;)